lib/doctor/check-ping.js

```diff
@@ -4,9 +4,8 @@ var ping = require('../ping.js')
 function checkPing (cb) {
   var tracker = log.newItem('checkPing', 1)
   tracker.info('checkPing', 'Pinging registry')
-  ping({}, true, function (err, pong) {
-    tracker.finish()
```

Comment:
I think we still should be finishing the tracker. While it doesn't matter when running `npm ping` it would help actually mark the step as completed for `npm doctor`.